# <font color=darkblue>CE 4363/5363 Groundwater Hydrology <br> Spring 2023 Exam 2</font>

**LAST NAME, FIRST NAME**

**R00000000**

___

### Purpose : 
Demonstrate ability to apply groundwater hydrology principles to aquifers

### Additional Instructions

1. The test is intended to be completed by extracting the problem statements and inserting worked answers, including computations into a document, saving that document as a PDF file, and uploading the completed exam to Blackboard.  
2. Show your work; handwritten is OK, but must be neat, organized, and legible (no brain vomit)
3. Cite reference sources used to support your work

**CE 4363** Complete problems 1-10<br>
**CE 5363** Complete problems 1-15<br>

<hr>
<hr>

## Question 1 (3 pts.)

Distinguish between the capabilities of ModelMUSE and MODFLOW.

<hr>
<hr>

## Question 2 (4 pts.)

List four potential sources of groundwater contamination.

<hr>
<hr>

## Question 3 (3 pts.)

Distinguish between calibration and validation for a numerical groundwater computer model.

<hr>
<hr>

## Question 4 (5 pts.)

Does the presence of a man-made chemical in groundwater make it unusable? Explain your answer in terms of regulatory or technical issues.

<hr>
<hr>

## Question 5 (5 pts.)

Explain how geophysical application of electrical resistivity can be used to distinguish between clay and sand, and between dry and saturated clay.

<hr>
<hr>

## Question 6 (5 pts.)

Explain the concept of safe yield for an aquifer. Why is it hard to define? Why must we try to define it?

<hr>
<hr>

## Question 7 (15 pts.)

A confined aquifer system is conceptualized as pictured below.

![](aquiferSystem.png)

Water enters the system in the recharge area and exits as a spring.

Determine:
1. The piezometric head in the well located at $M$ (express the head in terms of the other variables indicated on the sketch)
2. Specify the conditions for the well to become a flowing (artesian) well (water flows from the well without pumping).

<hr>
<hr>

## Question 8 (20 pts.)

A 4-cell aquifer model is conceptualized in the figure below.

![](aquiferSystem1D.png)

The width of the aquifer strip is 3.0 km; the length of each cell is 5 km.  The recharge rates for the aquifer strip is $400~\frac{mm}{yr}$, $300~\frac{mm}{yr}$,$300~\frac{mm}{yr}$, $200~\frac{mm}{yr}$ in Cells 1,2,3, and 4, respectively.  The water level in the river is maintained at a constant elevation of $160~m$ above the horizontal impervious bottom.  The hydraulic conductivity in Cells 1 and 2 is $30~\frac{m}{d}$, while in Cells 3 and 4 it is $60~\frac{m}{d}$.

Using this conceptual model, determine:

1. Write the steady-flow balance equations to estimate average water levels in the four cells.
2. Solve these equations to determine the average elevations in the 4 cells without pumping.
3. Solve these equations when pumping takes place in Cells 2 and 3 at rates of $4\times10^{6}~\frac{m^3}{yr}$ and $7\times10^{6}~\frac{m^3}{yr}$, respectively.

### Reference(s)

1. [Bear (1979). *Hydraulics of Groundwater* McGraw-Hill (Chapter 10: Modeling of Aquifer Systems)](ce-4363-webroot/3-Readings/BearChapter10.pdf) (Eq. 10-9) is useful.

<hr>
<hr>

## Question 9 (15 pts.)

A one-dimensional soil column experiment was performed to study the effects of dispersion on solutes in a soil sample. The soil column was 0.30 m long, and the actual flow velocity was set at 6.0 cm/hr. The dispersivity of the solute was found to be 4.5 cm. Molecular diffusion was neglected. A conservative tracer, which was initially absent in the soil column, was introduced to the influent end of the column at time zero at a concentration of 50 mg/L. 

Determine:
1. The concentration of the tracer in the column effluent in mg/L at time of 4.5 hr after injection began.

<hr>
<hr>

## Question 10 (20 pts.)

The figure is a plan view of a confined aquifer showing two contaminated zones, 1 and 2. The aquifer has a saturated thickness of 80 ft, hydraulic conductivity of 42 ft/d, and the regional hydraulic gradient is 0.0075 from right to left as shown. A pumping well with a flow rate of 170 gpm is planned to run continuously at the location shown, and form a stable (equilibrium) capture zone. You may assume that the 2500 ft distance from the well to zone 1 is long enough that the capture zone has reached its maximum width.

![](plumeria.png)

Determine:

1. If contamination from zone 1 find its way to the pumping well? Provide support for your answer with a capture zone analysis or contaminant simulation model.
2. If contamination from zone 2 find its way to the pumping well? Provide support for your answer with a capture zone analysis or contaminant simulation model.

### References

1. [Javandel,I., and Tsang,C-F. (1986) "Capture Zone Type Curves: A Tool for Aquifer Cleanup" *Groundwater* Vol. 24 No. 5 pp. 616-625](http://54.243.252.9/ce-4363-webroot/3-Readings/Capture.PDF)

<hr>

## CE 5363 Question 11 (20 pts.)

A saturated aquifer sample core of diameter 2.54 cm and length of 6 cm weighs 63 grams.  After drying the sample weighs 53 grams.  The core sample was placed into an permeameter and exposed to a unit hydraulic gradient.  The measured flowrate was 25.4 milliliters/second.

The water levels in three wells in the same aquifer were measured in meters above MSL.  The levels were: Well MW-1: 83.1 m; Well MW-2: 84.6 m; Well MW-3: 83.9 m.  

Well MW-2 is located 1km due north of Well MW-1, and Well MW-3 is located 700 meters Northeast of Well MW-1.

Determine:
1. Estimate the porosity of the aquifer.
2. Estimate the hydraulic conductivity of the aquifer.
3. Sketch the relative positions of the three wells.  Use MW-1 as the origin of the local (your sketch) coordinate system, and North is to the top of your sketch.
4. Determine the magniture and direction of the hydraulic gradient in the aquifer monitored by the three wells.  Indicate the hydraulic gradient on your sketch (direction and magnitude)
5. Estimate the concentration of a conservative constituient (contaminant) at a receptor 4 km away from MW-1 on a flowline that passes through MW-1 using the Ogata-Banks  model, assuming MW-1 is at the source location, and the source concentration is 1000 mg/l. Use one-tenth of the path length as the aquifer dispersivity (recall dispersion coefficient is the product of dispersivity and average linear velocity).  Calculate the receptor concentration from 0 days after release to 1000 days after release in 10 day increments, and make a concentration history plot for the receptor location.
6. An estimate the time from release until the concentration at the receptor is 500 mg/L.

### References

1. [Ogata, A., & Banks, R. B. (1961). A solution of the differential equation of longitudinal dispersion in porous media: fluid movement in earth materials. US Government Printing Office. (link is to an on-line calculator)](https://www.groundwatersoftware.com/calculator_5_ogata_banks.htm)

 
```
# Ogata-Banks Solution
# v  : velocity
# L  : distance downstream of source
# t  : time
# D  : dispersion coefficient
# Co : source concentration


def conc(L,t,Co,v,D):
    import math
    conc=(Co/2.0)*(math.erfc((L - v*t)/(2.0*math.sqrt(D*t))+math.exp((v*L)/(D))*math.erfc((L + v*t)/(2.0*math.sqrt(D*t)))))
    return(conc)
    
# Example pg 415 (without truncating terms as in Fetter)
v = 2.6e-7
L = 15.
t = 3.15e7
D = 2.4e-7
Co = 725.

print("C at L=:",L," meters is: ",round(conc(L,t,Co,v,D),3)," mg/L")  
```

<hr>

In [28]:
# Ogata-Banks Solution
# v  : velocity
# L  : distance downstream of source
# t  : time
# D  : dispersion coefficient
# Co : source concentration


def conc(L,t,Co,v,D):
    import math
    conc=(Co/2.0)*(math.erfc((L - v*t)/(2.0*math.sqrt(D*t))+math.exp((v*L)/(D))*math.erfc((L + v*t)/(2.0*math.sqrt(D*t)))))
    return(conc)

# Example pg 415 
v = 19.6
L = 4000.
t = 3.15e7
D = 78520
Co = 1000.

#print("C at L=:",L," meters is: ",round(conc(L,t,Co,v,D),3)," mg/L")                                                                                         
time = []
C = []

for i in range(100):
    time.append(float(i+1)*10)
    C.append(conc(L,time[i],Co,v,D))
for i in range(100):
    print(time[i],C[i])

10.0 1.1881791764868597
20.0 18.46520393039024
30.0 44.141620144060454
40.0 66.90788645150504
50.0 85.70748953091417
60.0 101.65519039194187
70.0 115.78094462816853
80.0 128.7795143414557
90.0 141.09025655156083
100.0 152.98801124824092
110.0 164.6450879279649
120.0 176.1696811906463
130.0 187.62929447198616
140.0 199.0651489252584
150.0 210.5012057233965
160.0 221.949933464867
170.0 233.41607319077963
180.0 244.89914626821246
190.0 256.39515643859033
200.0 267.8977646649691
210.0 279.3991122287687
220.0 290.89040471123604
230.0 302.36233053337196
240.0 313.80536311605994
250.0 325.20997989825304
260.0 336.5668211041354
270.0 347.86680427442775
280.0 359.1012059368546
290.0 370.2617186126621
300.0 381.34048914808795
310.0 392.330142804484
320.0 403.2237964306744
330.0 414.0150632384069
340.0 424.69805111394953
350.0 435.26735596313085
360.0 445.71805126021724
370.0 456.04567472292325
380.0 466.2462128454362
390.0 476.31608387363394
400.0 486.25211969091833
410.0 496.0515469915033
420.0

## CE 5363 Question 12 (20 pts.)

The drawing below is a profile through an unconfined aquifer with two drains located left and right of the aquifer.  Derive the equation that relates head in the aquifer to the recharge ($N$) and the position ($x$).  The water level in the left drain is fixed at elevation $h_L = 10.0$ meters.  The elevation of the land surface is $D=18.5$ meters.  If the right drain is one kilometer from the left drain and the desired minimum depth to water is $d_{min} = 1.0$ meter, find the required elevation of water, $h_R$, in the right drain.  The recharge rate is $N=0.001 \frac{m}{day}$, the hydraulic conductivity is $K=1.0 \frac{m}{day}$

![](Unconfined2Ditch.png)

<hr>

## CE 5363 Question 13 (20 pts.)

Given the four-cell conceptualization of an aquifer with the properties listed below 

![](4cellmodel.png)

Determine:

1. Write the steady-flow balance equations for average water levels in the four cells
2. Solve these equations to determine if the aquifer is a net supplier of water to the river or if the river is a supplier of water to the aquifer.
3. Write the steady-state balance equations for average concentration of a conservative tracer in the four cells.
4. If the river concentration of this tracer is 1,000 mg/L, calculate the steady-state concentrations in the four cells.

### Reference(s)

1. [Bear (1979). *Hydraulics of Groundwater* McGraw-Hill (Chapter 10: Modeling of Aquifer Systems)](ce-4363-webroot/3-Readings/BearChapter10.pdf) (Eq. 10-9) is useful, also (Eq. 10-11).  

<hr>
<hr>

## CE 5363 Question 14 (20 pts.)

An artifical recharge pond is located in a sandy area (Kv = 10 m/day; porosity = 30%).
The pond covers a large area. A flood wave fills the pond rapidly to a water level of 5.5 meters above the sandy bottom.
By controlling the inflow and outflow the water level is maintained at this elevation. 
During previous operations, the bottom of the pond was covered by a layer of fine material 0.5 meters thick with Kv = 0.1 m/day and a porosity of 50%.

Assuming infiltration is vertically downward only, determine the position of the wetting front from the recharge pond into an initially dry soil as a function of time (derive an equation). 
Assume a very deep water table.   What is the infiltration after a long time?

<hr>
<hr>